### PR TITLE
Github Actions using VPN In Staging

### PR DESCRIPTION
**Pull request by ben851**
## What happens when your PR merges?
All github actions that require connectivity to EKS in staging will now use openvpn to connect to the cluster instead of running on self-hosted github actions.

## What are you changing?
- [ ] Releasing a new version of Notify
- [x] Changing kubernetes configuration

## Provide some background on the changes
Working on stabilizing and simplifying our workflows.

## If you are releasing a new version of Notify, what components are you updating
- [ ] API
- [ ] Admin
- [ ] Documentation
- [ ] Document download API

## Checklist if making changes to Kubernetes:
- [x] I know how to get kubectl credentials in case it catches on fire

## After merging this PR
- [ ] I have verified that the tests / deployment actions succeeded
- [ ] I have verified that any affected pods were restarted successfully
- [ ] I have verified that I can still log into [Notify production](https://notification.canada.ca)
- [ ] I have verified that the smoke tests still pass on production
- [ ] I have communicated the release in the #notify Slack channel.

### CODE REVIEW

File: .github/workflows/config/staging.ovpn
Comment (ben851) [inline .github/workflows/config/staging.ovpn:1]: Not happy that this has to live in code, but it looks like it's the only way at the moment. This does not contain the private key, so it should be secure.

File: .github/workflows/admin-rollout-k8s-staging.yaml [hunk 1]
```diff
@@ -44,6 +44,17 @@ jobs:
...
+      - name: Install OpenVPN
+        run: |
+          sudo apt update
+          sudo apt install -y openvpn openvpn-systemd-resolved
+
+      - name: Connect to VPN
+        uses: "kota65535/github-openvpn-connect-action@v2"
+        with:
+          config_file: .github/workflows/config/staging.ovpn
+          client_key: ${{ secrets.STAGING_OVPN_CLIENT_KEY }}
```
Comment (jimleroyer) [inline .github/workflows/admin-rollout-k8s-staging.yaml:56]: Could we have this flow from our 1pw stuff? just wondering here, no blocker

**Review by jimleroyer — APPROVED** [review]
LGTM, left a question but no blocker

File: .github/workflows/admin-rollout-k8s-staging.yaml (hunk 1 above)
Comment (ben851) [reply .github/workflows/admin-rollout-k8s-staging.yaml:56]: hmm that's possible - I like that! Let me look into it!

Comment (ben851) [reply .github/workflows/admin-rollout-k8s-staging.yaml]: Yup - that works! Thanks for the idea!

**Pull request merged**